draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -106,7 +106,31 @@
 	"Proof-of-Possession Key Semantics for JSON Web Tokens (JWTs)" <xref target="RFC7800"/>,
 	but using CBOR <xref target="RFC7049"/> and CWTs <xref target="RFC8392"/>
 	rather than JSON <xref target="RFC8259"/> and JWTs <xref target="JWT"/>.
-     </t>
+     </t>>> Section 6
+>>
```

Comment:
">>" looks strange

Reply:
ah, work in progress

Reply:
This is really weird, my local copy of this branch does not have these syntax errors and copies of mail discussion, while if I load the remote branch, they are indeed there. I've no idea how this happened.